apps/api/v2/src/ee/bookings/2024-08-13/controllers/e2e/user-bookings.e2e-spec.ts

```diff
@@ -1936,292 +1938,465 @@ describe("Bookings Endpoints 2024-08-13", () => {
+        const timeZone = "Europe/London";
+        const attendee = await userRepositoryFixture.create({
+          email,
+          username,
+          locale,
+          timeZone,
+        });
 
-        const bookingBody: CreateBookingInput_2024_08_13 = {
-          start: new Date(Date.UTC(2040, 0, 9, 13, 0, 0)).toISOString(),
-          eventTypeId: eventTypeWithAllLocationsId,
-          attendee: {
-            name: "Mr Proper",
-            email: "mr_proper@gmail.com",
-            timeZone: "Europe/Rome",
-            language: "it",
+        const booking = await bookingsRepositoryFixture.create({
+          uid: `booking-uid-${randomString(10)}`,
+          title: "booking title",
+          startTime: "2060-09-05T11:00:00.000Z",
+          endTime: "2060-09-05T12:00:00.000Z",
+          eventType: {
+            connect: {
+              id: eventTypeId,
+            },
           },
-          location: {
-            type: "attendeePhone",
-            phone: attendeePhone,
+          status: "ACCEPTED",
+          metadata: {},
+          responses: "null",
+          user: {
+            connect: {
+              id: user.id,
+            },
           },
-        };
+          attendees: {
+            create: {
+              email: email,
+              name: username,
+              timeZone,
+              locale,
+            },
+          },
+        });
+        const bookingUid = booking.uid;
+        const bookingId = booking.id;
 
-        const bookingResponse = await request(app.getHttpServer())
-          .post("/v2/bookings")
-          .send(bookingBody)
-          .set(CAL_API_VERSION_HEADER, VERSION_2024_08_13)
-          .expect(201);
+        it("can update location to type address", async () => {
+          const updatedBookingBody: UpdateBookingInput_2024_08_13 = {
+            location: {
+              type: "address",
+            },
+          };
 
-        const bookingResponseBody: CreateBookingOutput_2024_08_13 = bookingResponse.body;
-        const createdBooking = bookingResponseBody.data;
-        expect(createdBooking).toHaveProperty("id");
+          const updatedBookingResponse = await request(app.getHttpServer())
+            .patch(`/v2/bookings/${bookingUid}`)
+            .send(updatedBookingBody)
+            .set(CAL_API_VERSION_HEADER, VERSION_2024_08_13)
+            .expect(201);
 
-        if (responseDataIsBooking(createdBooking)) {
-          expect(createdBooking.location).toEqual(attendeePhone);
-          await bookingsRepositoryFixture.deleteById(createdBooking.id);
-        } else {
-          throw new Error("Unexpected response data type");
-        }
-      });
+          const updatedBookingResponseBody: GetBookingOutput_2024_08_13 = updatedBookingResponse.body;
+          const updatedBooking = updatedBookingResponseBody.data;
+          expect(updatedBooking).toHaveProperty("id");
+          expect(createdBooking.location).toEqual(address);
```

Comment:
We should test updatet booking location `expect(updatedBooking.location).toEqual(address);` not `expect(createdBooking.location)` - this needs to be updated in the new tests.